README.md

```diff
@@ -105,7 +112,14 @@ Hint: To create S3 buckets from the command line, use `aws s3 mb --region REGION
 
 	7) Place the downloaded JSON file into the project root as `application_secrets.json`
     
-	8) Run the command `cat /path/to/application_secrets.json | --secret-name scripts/set_secret $GOOGLE_APPLICATION_SECRETS_SECRETS_NAME`
+    8) In `environment.local`, set the environment variable `GOOGLE_APPLICATION_SECRETS` to the path of the downloaded 
```

Comment:
`GOOGLE_APPLICATION_SECRETS` has also been nuked.

`cat /path/to/application_secrets.json | scripts/set_secret.py --secret-name $GOOGLE_APPLICATION_SECRETS_SECRETS_NAME`